docs/proposals/apb-state-support.md

```diff
@@ -0,0 +1,60 @@
+# Abstract
+
+This proposal will outline a simple and secure method for allowing APB developers to set state during a APB
+action and then be able to access that state during a subsequent APB action.
+
+## Actions
+   - provision
+   - deprovision
+   - update
+   - bind 
+   - unbind
+   
+   
+## Problem Description
+
+APBs are stateless. All state is managed for them by the broker. Currently the broker passes in parameters specified
+during a request to the catalog. It also passes in some additional parameters such as the namespace etc, additionally 
+we also pass in credentials created during the provision. While this is useful, there is no mechanism for an APB to store 
+and access data across actions without exposing it to the end user. We want to avoid APB developers working around this 
+limitation by doing suboptimal things such as storing extra data in the credentials or by creating a ConfigMap in the users 
+namespace as both of these expose the data to the user and as the namespace is controlled by the user, it is naturally 
+not trustworthy or reliable.
+
+
+
+# Proposed Solution
+
+## APB module for handling state
```

Comment:
Can we rename `bundle contract for handling state`? A minor nit right now because we don't have other modules. I just think this will need to be supported more broadly by the bundle/broker contract that @djzager and @mhrivnak have been working on.

Reply:
It makes sense to have this section detail the ansible module for handling state in the `ansible-asb-modules`.

Reply:
I am a bit uncertain here. Should I just replace the heading or all instances of APB?